### PR TITLE
Fix Salaries and Rankouts

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

File: core/src/database/franchise/player/player.model.ts [hunk 1]
```diff
@@ -26,7 +26,13 @@ export class Player extends BaseModel {
...
     skillGroupId: number;
 
-    @Column({type: "numeric"})
+    @Column({
+        type: "numeric",
+        transformer: {
+            from: (value: string) => parseFloat(value),
+            to: (value: number) => value.toString(),
+        },
+    })
```
Comment (zachstence) [inline core/src/database/franchise/player/player.model.ts:35]: Why is this necessary again? Why don't we need this on other number fields?

**Review by zachstence — CHANGES_REQUESTED** [review]

File: core/src/database/franchise/player/player.model.ts (hunk 1 above)
Comment (bwpikaard) [reply core/src/database/franchise/player/player.model.ts:35]: `numeric` is returned from typeorm as a string.

Comment (zachstence) [reply core/src/database/franchise/player/player.model.ts:35]: Why don't we change the column type in the DB?

Comment (bwpikaard) [reply core/src/database/franchise/player/player.model.ts:35]: Not sure. That's just what was there. I was going to change it to `double` or `float` but wasn't sure if there was a reason.

**Review by gankoji — DISMISSED** [review]

File: core/src/database/franchise/player/player.model.ts (hunk 1 above)
Comment (zachstence) [reply core/src/database/franchise/player/player.model.ts:35]: Lets change the column type

**Review by zachstence — APPROVED** [review]
